.github/workflows/semgrep.yml

```diff
@@ -21,5 +20,5 @@ jobs:
     container:
       image: returntocorp/semgrep
```

Comment:
Did you mean to change this to semgrep/semgrep?

Reply:
Yes, pushed a new commit.